pkgs/development/compilers/lessc/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
     description = "LESS to CSS compiler";
     homepage = http://lesscss.org/;
     license = licenses.asl20;
-    platforms = platforms.linux;
+    platforms = platforms.linux ++ platforms.darwin;
```

Comment:
@joehenry087 no; the channel moves forward once the build farm has churned through all packages and tests. Until then you can just build from your local checkout (substitutes should be copied to the cache once they are completed and so will be available to you, even if the channel hasn't moved ahead yet).